tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -756,3 +772,58 @@ func newTidbClusterConfig(cfg *tests.Config, ns, clusterName, password, tidbVers
 		ClusterVersion:         tidbVersion,
 	}
 }
+
+func newTidbMonitor(name, namespace string, tc *v1alpha1.TidbCluster, grafanaEnabled, persist bool) *v1alpha1.TidbMonitor {
```

Comment:
@aylei created a fixture pkg (https://github.com/pingcap/tidb-operator/tree/master/tests/pkg/fixture), move it to this pkg?

the old `newTidbClusterConfig` function in this file can be removed when we abandoned helm entirely.